cmd/broker.go

```diff
@@ -0,0 +1,211 @@
+	Run: func(cmd *cobra.Command, args []string) {
+		bootstrapBroker()
+	},
+}
+
+func init() {
+	brokerCmd.PersistentFlags().StringVarP(&brokerName, "name", "n", "openshift-automation-service-broker", "Name of Automation Broker instance")
+	rootCmd.AddCommand(brokerCmd)
+
+	brokerCmd.AddCommand(brokerCatalogCmd)
+	brokerCmd.AddCommand(brokerBootstrapCmd)
+}
+
+func listBrokerCatalog() {
+	log.Debugf("func::listBrokerCatalog()")
+	kube, err := clients.Kubernetes()
+	if err != nil {
+		log.Errorf("Failed to connect to cluster: %v", err)
+		return
+	}
+
+	brokerRoute, err := getBrokerRoute(brokerName)
+	if err != nil {
+		log.Errorf("Failed to get broker route: %v", err)
+	}
+
+	osbConf := &osb.ClientConfiguration{
+		Name:                "automation-broker",
+		URL:                 brokerRoute,
+		APIVersion:          osb.LatestAPIVersion(),
+		TimeoutSeconds:      60,
+		EnableAlphaFeatures: false,
+		Insecure:            true,
+		AuthConfig: &osb.AuthConfig{
+			BearerConfig: &osb.BearerConfig{
+				Token: kube.ClientConfig.BearerToken,
+			},
+		},
+	}
+	osbClient, err := osb.NewClient(osbConf)
+	if err != nil {
+		log.Errorf("Failed to make osb client: %v", err)
+		return
+	}
+
+	services, err := osbClient.GetCatalog()
+	printServices(services.Services)
+	return
+}
+
+func bootstrapBroker() {
+	kube, err := clients.Kubernetes()
+	if err != nil {
+		log.Errorf("Failed to connect to cluster: %v", err)
+		return
+	}
+
+	// Get the broker route given brokerName
+	brokerRoute, err := getBrokerRoute(brokerName)
+	if err != nil {
+		log.Errorf("Failed to get broker route: %v", err)
+		return
+	}
+
+	// Create a new bootstrap request
+	req, err := http.NewRequest("POST", fmt.Sprintf("%v/v2/bootstrap", brokerRoute), nil)
+	if err != nil {
+		log.Errorf("Failed to create request: %v", err)
+		return
+	}
+
+	// Set Bearer token Auth
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", kube.ClientConfig.BearerToken))
+
+	// Set TLS settings
+	cfg := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+	http.DefaultClient.Transport = &http.Transport{
+		TLSClientConfig: cfg,
+	}
+
+	// Do bootstrap request
+	fmt.Printf("Attempting to contact the broker at %v/v2/bootstrap...\n", brokerRoute)
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		log.Errorf("Failed to get response: %v", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		log.Errorf("Failed to bootstrap the broker. Expected status 200, got: %v", resp.StatusCode)
+		return
+	}
+
+	// Unmarshal response
+	jsonBoot, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Errorf("Failed to read response body: %v", err)
+	}
+
+	bootResp := bootstrapResponse{}
+	err = json.Unmarshal(jsonBoot, &bootResp)
+	if err != nil {
+		log.Errorf("Failed to unmarshal response: %v", err)
+	}
+
+	fmt.Printf("Successfully bootstrapped broker [%v]\n", brokerName)
+	fmt.Printf("Found %v specs out of %v total images.\n", bootResp.SpecCount, bootResp.ImageCount)
```

Comment:
I'd prefer changing this to specify the the broker was doing the work:
```
Broker found 10 specs out of 83 total images.
or
Broker loaded 10 bundle specs from 83 images.
or
Broker searched 83 images and found 10 bundle specs.
```

Reply:
Good feedback.